kukur/source/piwebapi_da/piwebapi_da.py

```diff
@@ -184,9 +184,13 @@ def get_metadata(self, selector: SeriesSelector) -> Metadata:
             session, self.__request_properties, data_archive
         )
 
-        point = response.json()["Items"][0]
+        items = response.json()["Items"]
+        if len(items) == 0:
+            raise InvalidDataError(
+                "No metadata found for the providade series selector"
```

Comment:
```suggestion
                "Series not found"
```

If we cannot find metadata, then it means the series does not exist. That will be a more useful error message to a user, otherwise it looks like we cannot find the metadata, rather than that the series they asked does not exist.